src/lib/slack/commands/ThingsBotCommand.js

```diff
@@ -19,18 +19,18 @@ const ThingsBotCommand = {
             const deviceList = await getDeviceList(this.installedSmartAppId);
 
             if (!deviceList) {
-                console.error('ERROR ** Failed to pull valid device list in deviceCommandsBlock getter')
+                console.error('ERROR ** Failed to pull valid device list in deviceCommandsBlock getter');
                 return {
                     "response_type": "ephemeral",
                     "text": "Uh oh, there was a problem pulling your device list. Please try again."
                 };
-            } else if (deviceList.length == 0) {
+            } else if (deviceList.length === 0) {
                 return {
                     "response_type": "ephemeral",
                     "text": "Oops, looks like you don't have any devices on your SmartThings account! Please add a device and then try again."
                 };
             }
-            
```

Comment:
Thanks. I think Sublime Text is more on top of stripping unnecessary whitespace. I'm tentatively using VS Code now and it doesn't seem to be as good about that. Also Sublime shows whitespace when you select lines which is really helpful. Code only has a global option to always show it or never show it so I have that disabled because I don't like the clutter always there

Reply:
Just WebStorm things :)